packages/internal-test-helpers/lib/module-for.js

```diff
@@ -41,9 +43,13 @@ export default function moduleFor(description, TestClass, ...mixins) {
   applyMixins(TestClass, mixins);
 
   let proto = TestClass.prototype;
-
   while (proto !== Object.prototype) {
-    Object.keys(proto).forEach(generateTest);
+    if(proto instanceof ObjectAccessorTestCase && !ENV.USES_ACCESSORS) {
```

Comment:
@rwjblue I converted over to your example. This is probably the only "weirdness" and needed [to support this functionality](https://github.com/thoov/ember.js/blob/2908e947ad9a8053ef9a2e468a0e0923fbb888d4/packages/internal-test-helpers/lib/test-groups.js#L19-L25). Let me know your thoughts on this.

Reply:
Do we ever set that ENV flag? I wonder why it exists? 🤔

Reply:
I just grepped and can't find any usage of it nor can I find when this was added when looking at the git history. Now looking at this deeper seems like testBoth has never been "testing" both and instead just doing dummy asserts on the non ember metal side. Do you think we should remove this behavior or attempt to support it?

Reply:
Kill it with 🔥

Reply:
![simpsons-fire](https://user-images.githubusercontent.com/166909/34695517-00ca0fae-f480-11e7-9ff2-e9cf55296b08.gif)